### PR TITLE
Init e2e Test for Get Command and update presubmit.sh

### PR DESCRIPTION
**Pull request by PuneetPunamiya**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if functionality changed/added)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/main/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/main/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: api/test/cli/cli.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+	"time"
+
+	"github.com/Netflix/go-expect"
+	goexpect "github.com/Netflix/go-expect"
+	"github.com/hinshun/vt10x"
+	"gotest.tools/assert"
```
Comment (vinamra28) [inline api/test/cli/cli.go:14]: can't we use `"gotest.tools/v3/assert"` this package?

File: api/test/e2e/get/get_test.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+package info
+
+import (
+	"testing"
+
+	"github.com/AlecAivazis/survey/v2/terminal"
+	"github.com/Netflix/go-expect"
+	"github.com/tektoncd/hub/api/test/cli"
+	"gotest.tools/assert"
```
Comment (vinamra28) [inline api/test/e2e/get/get_test.go:11]: same as above

File: test/presubmit-tests.sh [hunk 3]
```diff
@@ -183,8 +198,13 @@ unit_tests() {
 }
 
 integration_tests() {
-  warn "No integration tests to run"
-  return 0
+  (
+    set -eu -o pipefail
+
+    cd "$API_DIR"
+    api-e2e || return 1
```
Comment (vinamra28) [inline test/presubmit-tests.sh:205]: should we add a makefile target for it?

File: test/e2e-test.sh [hunk 4]
```diff
@@ -0,0 +1,12 @@
+cd api
+
+failed=0
```
Comment (vinamra28) [inline test/e2e-test.sh:3]: can we add a shebang `#!/usr/bin/env bash` ?

Comment (vinamra28) [reply test/e2e-test.sh:3]: also the license

File: test/presubmit-tests.sh (hunk 3 above)
Comment (PuneetPunamiya) [reply test/presubmit-tests.sh:205]: Yep going to add it once the patch gets in and will address in a separate patch

File: test/e2e-test.sh (hunk 4 above)
Comment (PuneetPunamiya) [reply test/e2e-test.sh:3]: Yep will add it 👍🏻

File: api/test/e2e/get/get_test.go [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+					return err
+				}
+				if _, err := c.ExpectEOF(); err != nil {
+					return err
+				}
+
+				c.Close()
+				return nil
+			}})
+	})
```
Comment (piyush-garg) [inline api/test/e2e/get/get_test.go:67]: what are we asserting here ?

File: test/e2e-test.sh [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+if [[ -e ./bin/tkn-hub ]];then
+    export TEST_CLIENT_BINARY="${PWD}/bin/tkn-hub"
+else
+    go build -o tkn-hub github.com/tektoncd/hub/api/cmd/tkn-hub
+    echo "Go Build successfull"
+    export TEST_CLIENT_BINARY="${PWD}/tkn-hub"
+fi
+
+go test -v -count=1 -tags=e2e ./test/...
```
Comment (piyush-garg) [inline test/e2e-test.sh:37]: where this script is getting used?

**Comment by sm43** [comment]
@PuneetPunamiya could you address those comments and update the pr?

File: test/e2e-test.sh (hunk 6 above)
Comment (PuneetPunamiya) [reply test/e2e-test.sh]: This scipt is no more needed

File: api/test/e2e/get/get_test.go [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+	t.Run("Result for get interactive mode", func(t *testing.T) {
```
Comment (vinamra28) [inline api/test/e2e/get/get_test.go:100]: can we change the test title to something else as the command below is not interactive and can be misleading?

**Pull request merged**